SCons/Tool/ninja/ninja_daemon_build.py

```diff
@@ -53,6 +53,10 @@
     level=logging.DEBUG,
 )
 
+def log_error(msg):
+    logging.debug(msg)
```

Comment:
1. if it's an error, isn't `logging.error` more appropriate? 
2.  I know it's possible to configure logging to go both places, but if it needs to be log-level-dependent which goes where, I know it's not all that simple (multiple loggers, inheritance, etc. - may not be worth it).  The simple case is to just add `stream=sys.stderr` to the initialization in `basicConfig`.

Reply:
1. The logging levels are useful when the volume of logging is overwhelming. Currently (IMHO) the ninja daemon doesn't put out enough logging for the logging levels to really be useful. Not saying logging.error is not more appropriate or correct, just kind of meaningless in this context and use case. We can easily change it in the future it the logging become hard to manage.

2. Maybe one day if the logging situation calls for it. For now I think this is fine.